api_tests/src/image.spec.ts

```diff
@@ -0,0 +1,46 @@
+jest.setTimeout(120000);
+
+import { UploadImage, DeleteImage } from "lemmy-js-client";
+import { alpha, setupLogins, unfollowRemotes } from "./shared";
+import fs = require("fs");
+
+beforeAll(setupLogins);
+
+afterAll(() => {
+  unfollowRemotes(alpha);
+});
+
+test("Upload image and delete it", async () => {
+  // upload test image
+  // TODO: this doesnt require separate auth anymore (same for delete)
+  const upload_image = fs.readFileSync("test.png");
+  const upload_form: UploadImage = {
+    image: upload_image,
+  };
+  const upload = await alpha.uploadImage(upload_form);
+  console.log(upload);
+  expect(upload.files![0].file).toBeDefined();
+  expect(upload.files![0].delete_token).toBeDefined();
+  expect(upload.url).toBeDefined();
+  expect(upload.delete_url).toBeDefined();
+
+  // ensure that image download is working. theres probably a better way to do this
+  const downloadFileSync = require("download-file-sync");
```

Comment:
I'd still try to move this to the top as an import rather than a require. Try the `yarn add -D @types/download-file-sync` , but again that shouldn't be a requirement.

Reply:
Okay moving it to the top works, but still using const/require as it breaks with import. Also removed the todos.